server/test/serverTest.js

```diff
@@ -1,575 +1,661 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-unused-expressions */
```

Comment:
Was this because of the Chai `expect...` statements?

Reply:
yes

Reply:
I wonder if something like this would help: https://coderwall.com/p/zyi8gg/eslint-configuration-for-mocha-and-chai

Reply:
It looks like it would at a glance!

Reply:
That didn't seem to work but I added this to the `.eslintrc.json` file in the server directory and it does 
```
      "overrides": [{
        "files": [ "./test/*" ],
        "rules": {
          "no-unused-expressions": 0
        }
      }]
```
You mentioned that (I think) the app has a different file structure for tests but I still think this should be centralised so my inclination is to add this to the brave config repo along with another exception that covers that case, not sure what you think?

Reply:
OK cool. That works for me. I think right now it's only my Alert App code that isn't using the `test` directory structure and I can change that to match.